impls/go-channels/gozillions.go

```diff
@@ -0,0 +1,111 @@
+// Steve Phillips / elimisteve
+// 2017.03.12
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"sync"
+)
+
+func main() {
+	listenAddr := "127.0.0.1:" + parsePort()
```

Comment:
As per [the spec](https://github.com/dwrensha/zillions/blob/e3af5668b026073d7285a17ad7142b870454ad42/README.md#specification), the single command-line argument passed to your server is a full IP address -- not just a port.

I think this explains (2) and (3) in https://github.com/dwrensha/zillions/issues/3